.circleci/config.yml

```diff
@@ -329,7 +329,7 @@ jobs:
       # Cache the venv directory that contains dependencies
       - restore_cache:
           keys:
-            - cache-key-gpu-exp-112-394-{{ checksum "requirements-test.txt" }}-{{ checksum ".circleci/config.yml" }}
+            - cache-key-gpu-exp-112-394-{{ checksum "requirements-experimental.txt" }}-{{ checksum ".circleci/config.yml" }}
```

Comment:
make sure that we update the cache when the reference triton2.dev version is changed